pkg/pillar/cmd/zedrouter/appnumonunet.go

```diff
@@ -0,0 +1,185 @@
+// Copyright (c) 2021 Zededa, Inc.
```

Comment:
I would expect this file to be a clone of appnumallocator.go with the minimal modifications (the functions need the NI UUID) and nothing else, in order to avoid unique bugs.